sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/DigitalTwinsModelData.cs

```diff
@@ -29,6 +30,14 @@ public partial class DigitalTwinsModelData
         [CodeGenMember("UploadTime")]
         public DateTimeOffset? UploadedOn { get; }
 
+        /// <summary> A language map that contains the localized display names as specified in the model definition. </summary>
+        [CodeGenMember("DisplayName")]
+        public IReadOnlyDictionary<string, string> DisplayNameLanguageMap { get; }
```

Comment:
Are we changing this on all SDKs?

Reply:
FWIW, I don't disagree with the name change, but it came as a surprise. I don't recall hearing about it in the Azure SDK review...

Reply:
Tagging Zoltan now...